virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -161,6 +161,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //CNV-33125: Add CPU limits to the UI
 * Cluster admins can now enable CPU resource limits on a namespace in the {product-title} web console under *Overview* -> *Settings* -> *Preview features*.
 
+//CNV-22314: Safe memory overcommitment using `wasp-agent`
+* Cluster admins can now xref:../../virt/virtual_machines/virt-configuring-higher-vm-workload-density.adoc#virt-configuring-higher-vm-workload-density[use the `wasp-agent` tool to configure a higher VM workload density] in their clusters by overcommitting the amount of memory (RAM) and assigning swap resources to VM workloads.
```

Comment:
For accessibility reasons I'm not sure inline linking is the way to go. How about step out the link on a new line:

> For more information, see Configuring higher VM workload density.

This way when a customer clicks on the link it opens to a matching title.

Reply:
I'll leave it to you, but for accessibility reasons, inline links can be tricky.

![Uploading Screenshot from 2024-07-05 14-31-25.png…]()

Reply:
@dfitzmau  I've left the link inline, but slightly modified the link text to better correspond to the heading of the section it leads to.